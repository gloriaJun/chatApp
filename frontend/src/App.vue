<template>
  <v-app id="chat">
    <v-layout
      row
      align-center
      justify-center
    >
      <v-flex
        xs12
        sm6
      >
        <v-card
          class="chat-wrapper"
        >
          <router-view/>
        </v-card>
      </v-flex>
    </v-layout>
  </v-app>
</template>

<script>

export default {
  name: 'App',
};
</script>
