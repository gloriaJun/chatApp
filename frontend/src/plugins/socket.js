import Vue from 'vue';
import io from 'socket.io-client';

const socket = io(process.env.VUE_APP_SOCKET_URL);

Vue.prototype.$socket = socket;
