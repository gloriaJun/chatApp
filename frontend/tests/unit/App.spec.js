import { mount } from '@vue/test-utils';
import '@/plugins/vuetify';

import App from '@/App.vue';

describe('App.vue', () => {
  test('renders correctly', () => {
    const wrapper = mount(App);

    // console.log(wrapper.find('div'));
    expect(wrapper.element).toMatchSnapshot();
  });

  // it('tag check - router-view', () => {
  //   const wrapper = mount(App);
  //   expect(wrapper.html()).to.include('<router-view></router-view>');
  // });
});


// import { shallowMount } from '@vue/test-utils'
// import HelloWorld from '@/components/HelloWorld.vue'
//
// describe('HelloWorld.vue', () => {
//   it('renders props.msg when passed', () => {
//     const msg = 'new message'
//     const wrapper = shallowMount(HelloWorld, {
//       propsData: { msg }
//     })
//     expect(wrapper.text()).toMatch(msg)
//   })
// })
