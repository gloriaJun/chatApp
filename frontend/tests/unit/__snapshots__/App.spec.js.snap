// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue snapshot check 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="chat-app"
>
  <div
    class="application--wrap"
  >
    <router-view-stub />
  </div>
</div>
`;
